src/transport_picoquic.cpp

```diff
@@ -326,7 +326,7 @@ int pq_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode, void
     int ret = 0;
 
     if (transport == NULL) {
-        std::cerr << "picoquic transport was called with NULL transport" << std::endl;
+        std::cerr << "picoquic transport was called with NULL transport" << '\n';
```

Comment:
This one and below can go either way. Using '\n' over `std::endl` can be a gain if you don't need to flush the stream, although with `std::cerr` or `std::cout` they always flush anyway, so this isn't actually a gain. 

However, changing it does have the benefit of not ignoring a rule that may make a difference if another stream gets introduced into the codebase later, and using `\n` consistently is the recommendation.